src/main/java/org/apache/commons/cli/Util.java

```diff
@@ -43,8 +73,8 @@ static boolean isEmpty(final Object[] array) {
      * @param str The string to test.
      * @return Whether the given string is null or empty.
      */
-    static boolean isEmpty(final String str) {
-        return str == null || str.isEmpty();
+    public static boolean isEmpty(final CharSequence str) {
```

Comment:
Elements that are made public or protected should carry the `@since` Javadoc tag.